src/deploy/Network.ts

```diff
@@ -245,6 +243,30 @@ export async function deployNetworkComet(
     configuratorProxy = await deploymentManager.contract('configurator') as Configurator;
   }
 
+
+  if (shouldDeploy(contractsToDeploy.all, contractsToDeploy.rewards)) {
+    rewards = await deploymentManager.deploy<CometRewards, CometRewards__factory, [string]>(
+      'CometRewards.sol',
+      [adminSigner.address]
+    );
+
+    // Set the rewards config for Comet in CometRewards
+    debug(`Setting RewardConfig in CometRewards`);
+    await deploymentManager.asyncCallWithRetry(
+      (signer_) => wait(rewards.connect(adminSigner_ ?? signer_).setRewardConfig(cometProxy.address, rewardTokenAddress))
+    );
+
+    // Transfer ownership of CometRewards
+    debug(`Transferring ownership of CometRewards`);
+    await deploymentManager.asyncCallWithRetry(
+      (signer_) => wait(rewards.connect(adminSigner_ ?? signer_).transferGovernor(governor))
+    );
```

Comment:
Basically these are the 2 steps I should need for my refactor right?

Reply:
Yep